src/layout/BaseLayout.tsx

```diff
@@ -33,6 +33,9 @@ const BaseLayout: React.FC<BaseLayoutProps> = props => {
       <section className="main-header">
         <UniProtHeader isHomePage={isHomePage} isSearchPage={isSearchPage} />
       </section>
+      <section className="in-page-messages">
```

Comment:
Either the BaseLayout should alway be rendered, or this should be placed at a higher level (near the root of the app), because at the moment it is unmounted and remounted when browsing through the pages of the website and it's strange (appear/disappear, reactivate the apparition transition for already existing messages)

Reply:
Created https://www.ebi.ac.uk/panda/jira/browse/TRM-23983